DemoServer/AdditionalLanguages/nodejs/demo/staticIndexes/storeFieldsOnIndex/metadata.json

```diff
@@ -0,0 +1,228 @@
+                "",
+                "* Once the index is deployed, the server will start the _'indexing process'_,",
+                "  in which the index iterates over the documents, building a map between the terms derived from the document fields indexed and the actual documents that contain them.<br>",
+                "",
+                "* In the case of a Map-Reduce index, a second stage is applied where the mapped entries are reduced to some aggregated value.",
+                "",
+                "* After the first indexing run, the index will keep this map current without re-processing the entire data,",
+                "  and will only update the relevant details upon any update/change to any document related to the index."
+            ],
+            "Assets": [
+                //TODO
+                //{
+                //    "Title": "Sending Indexes to Server",
+                //    "Type" : "link",
+                //    "Url"  : "https://ravendb.net/docs/article-page/latest/nodejs/indexes/creating-and-deploying#sending-to-server"
+                //},
+                {
+                    "Title": "Deploying Indexes",
+                    "Type": "link",
+                    "Url": "https://ravendb.net/learn/inside-ravendb-book/reader/4.0/12-working-with-indexes#deploying-indexes"
+                },
+                {
+                    "Title": "Indexing Process",
+                    "Type": "link",
+                    "Url": "https://ravendb.net/docs/article-page/latest/nodejs/studio/database/indexes/indexes-overview#indexing-process"
+                }
+            ]
+        },
+        {
+            "Title": "Query for documents",
+            "Slug": "step-5",
+            "DescriptionLines": [
+                "* Query the 'Orders' collection using the defined index.",
+                "",
+                "* The resulting documents will include only orders that where made by 'companyId'."
+            ],
+            "Assets": [
+
+                {
+                    "Title": "Filtering Results - Basics Demo",
+                    "Type": "demo",
+                    "Url": "/queries/filtering-results-basics"
+                }
+            ]
+        },
+        {
+            "Title": "Project Results",
+            "Slug": "step-6",
+            "DescriptionLines": [
+                "* Use `selectFields()` to make the server return only that fields per document result instead of the whole document.<br>",
+                "  * If the field is _stored on the index_ - the server will fetch it from the index<br>",
+                "  * If the field is _not stored on the index_ - the server will fetch it from the document store",
+                "",
+                "* In this example:<br>",
+                "  * 'TotalItemsOrdered' is fetched from the index<br>",
+                "  *  'OrderedAt' if fetched from the document store"
```

Comment:
What about usage of `projectInto` ? (instead of selectFields)
https://ravendb.net/docs/article-page/5.3/nodejs/indexes/querying/projections#projectinto

Can it be mentioned ? as was done in the C# demo ?